### PR TITLE
Add release pleae configuration

### LINKED ISSUE #400: add release please configuration
reflection of https://github.com/microsoftgraph/msgraph-beta-sdk-java/issues/940 on this repository. We want:
- [ ] release please header file (yaml .github/release-please.yml)
- [ ] release please manifest (json .release-please-manifest.json)
- [ ] release please config (json release-please-config.json)
- [ ] release please workflow to trigger on main branch (.github/workflows/release-please.yml)
- [ ] install the [release please GitHub app](https://github.com/apps/release-please) to the repository
- [ ] changelog keep a changelog header and unreleased information to be removed
- [ ] contributing to be updated with commit formatting instructions
- [ ] version bump script to be deleted (and removed from the weekly generation process)
- [ ] files where the version needs to be updated to be ammended with the special comments for release please

### PR DESCRIPTION
**Pull request by shemogumbe**
fixes #400

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    ".": "1.4.0"
```
Comment (SilasKenneth) [inline .release-please-manifest.json:2]: Any reason why this is below of the version in `_version.py`?

Comment (baywet) [inline .release-please-manifest.json:2]:
```suggestion
    ".": "1.5.0"
```

File: release-please-config.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    ],
+    "release-type": "python",
+    "include-component-in-tag": false,
+    "include-v-in-tag": true,
+    "packages": {
+      ".": {
+        "package-name": "msgraph_beta",
+        "changelog-path": "CHANGELOG.md",
+        "extra-files": [
+          "README.md",
```
Comment (baywet) [inline release-please-config.json:17]:
```suggestion
```

File: release-please-config.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          "pyproject.toml",
+          "msgraph/_version.py"
```
Comment (baywet) [inline release-please-config.json:19]:
```suggestion
          "msgraph_beta/_version.py"
```

**Review by baywet — CHANGES_REQUESTED** [review]

**Review by baywet — DISMISSED** [review]

File: .release-please-manifest.json (hunk 1 above)
Comment (baywet) [reply .release-please-manifest.json]: I think he started from an old branch. Just solved that.

**Comment by SilasKenneth** [comment]
Hi @shemogumbe this looks good to me.
I also believe we should delete the scripts folder since we no longer need it.

**Review by baywet — APPROVED** [review]

**Review by SilasKenneth — APPROVED** [review]

**Pull request merged**